src/uk/gov/hmcts/pipeline/EnvironmentApprovals.groovy

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.pipeline
+
+class EnvironmentApprovals {
```

Comment:
Probably worth a test? Should be able to lift most of it from the TeamNames test?